docs/compute.md

```diff
@@ -80,8 +80,23 @@ To remove any passthrough devices from dom0:
 /opt/xensource/libexec/xen-cmdline --delete-dom0 xen-pciback.hide
 ```
 
+:::caution NVMe storage devices on Linux
+For NVMe storage devices, the Linux driver will try to allocate too many PCI MSI-X vectors, exceeding the number of extra IRQs allocated by Xen for a guest. Failing MSI-X setup might lead to very low performances on some buggy hardware if the driver cannot manage to fallback to legacy IRQs handling.
```

Comment:
"the Linux driver will try to allocate too many PCI MSI-X vectors"

Is this always true? It always tries to allocate too many of them?

Reply:
Yes, this is always true as the driver allocates as many vectors as the possible number of CPUs (not present ones) and this number of possible CPUs is always 64. This is because of the CPU topology in the ACPI table built by hvmloader.
I didn't want to go to deep in the details so I skipped this part...